patches/server/0069-jafama-fast-math.patch

```diff
@@ -0,0 +1,408 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Simon Gardling <titaniumtown@gmail.com>
+Date: Fri, 12 Feb 2021 15:18:15 -0500
+Subject: [PATCH] jafama fast math
+
+
+diff --git a/pom.xml b/pom.xml
+index 9402ec96f626e5e4f36f7dd678454f9efeb7e254..35845531dc7be489f67698f91ffd3cc7f5f1195c 100644
+--- a/pom.xml
++++ b/pom.xml
+@@ -182,6 +182,12 @@
+             <artifactId>commons-rng-core</artifactId>
+             <version>1.3</version>
+         </dependency>
++        <!-- Fast Math -->
```

Comment:
Don't you want to move it to our pom.xml patch for a better maintainability ?

Reply:
Ah yes! I forgot. I'll do that right now.

Reply:
done